string sender
bool centerSensorIn
bool leftSensorIn
bool rightSensorIn
